src/Microsoft.DotNet.Interactive.PowerShell.Tests/PowerShellKernelTests.cs

```diff
@@ -417,4 +417,74 @@ public async Task Powershell_CustomObject_Is_Formatted_Correctly(string mimeType
               .Should()
               .BeEquivalentHtmlTo(_tableOutputOfCustomObjectTest);
     }
+
+    [Fact]
+    public async Task PowerShell_objects_without_custom_formatters_use_native_formatting()
+    {
+        // Arrange
+        var kernel = CreateKernel(Language.PowerShell);
+        
+        // Act - Get-Process returns objects without custom formatters and should use PowerShell native formatting
+        var result = await kernel.SendAsync(new SubmitCode("Get-Process | Select-Object -First 1"));
+
+        // Assert - Should produce StandardOutputValueProduced (native PowerShell formatting) not DisplayedValueProduced
+        var outputs = result.Events.OfType<StandardOutputValueProduced>().ToList();
+        outputs.Should().NotBeEmpty("Get-Process output should use native PowerShell formatting");
+        
+        // The output should contain typical PowerShell table formatting headers
+        var allOutput = string.Join("", outputs.SelectMany(e => e.FormattedValues.Select(v => v.Value)));
+        allOutput.Should().NotBeEmpty();
```

Comment:
A more specific assertion to show that this is in fact table formatting would avoid some potential false positives here.